cli.go

```diff
@@ -140,7 +141,8 @@ func (cli *CLI) Run(args []string) int {
 	}
 
 	//Creating Events
-	events := eventRouting.NewEventRouting(cachingClient, loggingClient, statistic)
+	eventFilters := []eventRouting.EventFilter{eventRouting.HasIgnoreField, eventRouting.NotInCertainOrgs(*cfOrgs)}
```

Comment:
Filter should exclude event that match one of the item provided in the exclude_orgs comma separated list, not clear here.

Reply:
Don't see any issue here. We just pass the field as it is.